content/pages/education/opt-out-information-sharing.md

```diff
@@ -1,98 +1,54 @@
 ---
```

Comment:
most of these changes were pulled from [this](https://github.com/department-of-veterans-affairs/vets-website/pull/7947) content update